nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/serialization/AffectedComponentSet.java

```diff
@@ -80,6 +81,7 @@ public class AffectedComponentSet {
     private final Set<ControllerServiceNode> controllerServices = new HashSet<>();
     private final Set<ReportingTaskNode> reportingTasks = new HashSet<>();
     private final Set<ParameterProviderNode> parameterProviders = new HashSet<>();
+    private final Set<FlowRegistryClientNode> flowRegistryClients = new HashSet<>();
```

Comment:
The code here keeps track if Flow Registry Clients if ever they are added (but I don't think they ever should be). But we never do anything with them if they are added. I don't think we need to modify this class at all.

Reply:
Ah OK I missed that.